packages/x-data-grid-premium/src/tests/cellSelection.DataGridPremium.test.tsx

```diff
@@ -73,7 +73,7 @@ describe('<DataGridPremium /> - Cell selection', () => {
   it('should work with the paginated grid', () => {
     render(
       <TestDataGridSelection
-        initialState={{ pagination: { paginationModel: { page: 0, pageSize: 3 } } }}
+        initialState={{ pagination: { paginationModel: { page: 0, pageSize: 3 }, rowCount: 4 } }}
```

Comment:
```suggestion
        initialState={{ pagination: { paginationModel: { page: 0, pageSize: 3 }, rowCount: 0 } }}
```
I think that any value helps, it does not have to match. But it is fine even if you leave it like this